SIGS_AND_WGS.md

```diff
@@ -0,0 +1,25 @@
+# SIGs and WGs
+
+We follow the Kubernetes model for [SIGs] and [WGs].
+
+## [SIGs] (Special Interest Groups)
+
+Our [SIGs] define mission and scope via their charter, ownership of code via [OWNERS](https://www.kubernetes.dev/docs/guide/owners/), and define members and roles in community [sigs.yaml].
```

Comment:
sigs.yaml hyperlink in above sentence seem to be broken

![image](https://github.com/kubevirt/community/assets/156769580/225d8907-bc51-4194-ac2c-715b1be7e495)

Reply:
IIUC the link doesn't work in the preview since it's not merged yet - or am I mistaken?

Reply:
@dhiller The hyperlink sigs.yaml is broken in line #7 and in line #25, but where as in line #12, sigs.yaml hyperlink is working. So it seemed like the hyperlink in those two places is broken (probably because there isn't any URL attached to hyperlink? )

Reply:
Since I am using reference links, the reference for the link is defined in the bottom section in order to avoid repetitions. Please see [the markdown spec](https://daringfireball.net/projects/markdown/syntax#link) 

In line 12 I am referring to a specific line, therefore I avoided using the reference only.

Reply:
@dhiller Thank you for patiently explaining me about this and sharing the link. Now I understand the markdown better :)
By the way this file when opened from your [branch](https://github.com/dhiller/community/blob/wg-template/SIGS_AND_WGS.md) the links are working fine

Reply:
@chandramerla you're welcome :slightly_smiling_face: 

Thanks for your comment about my branch - good point, I should have included that in my comment.